### ISSUE TITLE
vip not implemented.

### ISSUE THREAD

**Issue by petersilva** [MetPX/sarrac#61]
the vip option is not implemented in sarrac.
```
peter@idefix:~/src/sarrac$ ./sr_cpost -c local_post.conf uthash.h
2019-06-17 17:16:09,287 [WARNING] info: vip option not implemented, ignored.
2019-06-17 17:16:09,287 [DEBUG] argument after substitutions: True

```

**Comment by petersilva** [comment]
commits: 769f0f8c90e4d2da224b0e7a840fe73844c49d3b and 849271956ffe108a82eaa4b7a6f7ca52f65d6752 
implemented this.

**Comment by petersilva** [comment]
included in 2.19.06b3
